modules/admixerBidAdapter.js

```diff
@@ -23,8 +23,8 @@ export const spec = {
    * Determines whether or not the given bid request is valid.
    */
   isBidRequestValid: function (bid) {
-    return bid.bidder === 'admixerwl'
-      ? !!bid.params.clientId && !!bid.params.endpointId
+    return bid.bidder === 'rtbstack'
+      ? !!bid.params.endpointId
```

Comment:
So you have a bunch of active aliases but you only want one of them to pass validity checks,?

Reply:
No, all aliases must have zone which is checked in the right part of operator, only rtbstack requires other params